README.md

```diff
@@ -1,6 +1,6 @@
 # zarr-extensions
 
-This repository contains the specifications for Zarr extensions for [Zarr version 3](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html).
+This repository contains the specifications for Zarr extensions as well as a catalog of registered attributes for [Zarr version 3](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html).
```

Comment:
I could see combining these two lines into:

> This repository is the normative source of registered names for the Zarr v3 specification. This includes Zarr extensions as well as a catalog of registered attributes for [Zarr version 3](https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html).

However, this does start to raise questions about the name of the repository ;)